source/administration/config-settings.rst

```diff
@@ -43,6 +43,8 @@ Site URL
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The URL, including port number and protocol, from which users will access Mattermost. Leave blank to automatically configure based on incoming traffic.
```

Comment:
Sorry @JeffSchering not sure what you mean by a URL is more than just the protocol+domain+port - I think it is the URL to the root page of the site? Do you mean Site URL doesn't make it clear that team shouldn't be included?